RoktDemo/ContentView.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ContentView.swift
+//  RoktDemo
+//
+//  Copyright 2020 Rokt Pte Ltd
+//
+//  Licensed under the Rokt Software Development Kit (SDK) Terms of Use
+//  Version 2.0 (the "License");
+//
+//  You may not use this file except in compliance with the License.
+//
+//  You may obtain a copy of the License at https://rokt.com/sdk-license-2-0/
+
+import SwiftUI
+
+struct ContentView: View {
+    var body: some View {
+        Text("Hello, world!")
```

Comment:
!!!!

Reply:
Yeah :D That's all it does right now. Will update it soon